### ISSUE TITLE
Items in social-links.html are listed as list items but not in a `<ul>` or `<ol>`

### ISSUE THREAD

**Issue by SQLJames** [razonyang/hugo-theme-bootstrap#183]
This seems to not be needed, removing the Li tags from each item on the social-links.html layout seems to keep the same layout and functionality. This applies to all links in this page.

```
  <li class="nav-item">
    <a class="nav-link social-link" target="_blank" href="{{ .Permalink }}" title="RSS" rel="noopener noreferrer">
      <i class="fas fa-fw fa-2x fa-rss"></i>
    </a>
  </li>
  ```
  vs 

```
    <a class="nav-link social-link" target="_blank" href="{{ .Permalink }}" title="RSS" rel="noopener noreferrer">
      <i class="fas fa-fw fa-2x fa-rss"></i>
    </a>
  ```

**Comment by SQLJames** [comment]
https://github.com/razonyang/hugo-theme-bootstrap/pull/184

**Comment by razonyang** [comment]
Closed by #184
